lib/project_types/extension/messages/messages.rb

```diff
@@ -173,6 +177,7 @@ module Messages
       errors: {
         unknown_type: "Unknown extension type %s",
         package_not_found: "`%s` package not found.",
+        missing_api_key: "Missing api_key.",
```

Comment:
I assume this is good enough based on https://github.com/Shopify/shopify-cli/blob/673d77da2f9f6d6176df775bca102f8e8fe281f5/lib/project_types/extension/messages/messages.rb#L111

Reply:
Let's loop in @MeredithCastile in case she wants to revisit it.